Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -18,7 +21,10 @@ module Homebrew
   def linkage
     ARGV.kegs.each do |keg|
       ohai "Checking #{keg.name} linkage" if ARGV.kegs.size > 1
-      result = LinkageChecker.new(keg)
+      database_cache = DatabaseCache.new("linkage")
```

Comment:
- Should this cache object be created every time we go through the loop?
- Pedantry but `:linkage` feels like a nicer argument; makes clear it's a type rather than e.g. a filename